### ISSUE TITLE
status 1

### ISSUE THREAD

**Issue by pmatcovich** [minbrowser/min#2106]
- **Min Version: v1.25.1
- **Operating system: Mint 21 Vanessa

## Expected Behavior

To be able to install ProtonVPN & Chromium

## Actual Behavior
Chromium
E: min: installed min package post-installation script subprocess returned error exit status 1

ProtonVPN
Run sudo xdg-settings set default-web-browser min.desktop && sudo update-alterna
tives --config x-www-browser to set Min as your default browser.
ln: failed to create symbolic link '/usr/bin/min': File exists
dpkg: error processing package min (--configure):
 installed min package post-installation script subprocess returned error exit s
tatus 1

## To Reproduce

Try to install ProtonVPN or Chromium

Troubleshooting included:

Set Min as the default browser in Default Applications ((Settings Manager)

sudo xdg-settings set default-web-browser min.desktop

Neither worked.

I was trying to insatll Chromium so I could list it as an alternative browser.

Your thoughts on this would be appreciated.

Thanks and Enjoy,

PAM

**Comment by PalmerAL** [comment]
Sorry about that. This is the same issue as #2102 I think; we changed the installer to add a symlink, and apparently broke installation when Min is already installed. If you do `rm /usr/bin/min` and then install, I think it should work.

**Comment by pmatcovich** [comment]
Yep, that did the trick! I greatly appreciate yout help. Unfortunatly, when I turned off my and turn it back on,  ProtonVPN hangs at first screen "Secure internet anywhere". Worse, I lose internet connectivity for browser and email, whether ProtonVPN is invoked or not.  I am on my laptop now. Network log shows " ... unknown host. " I am pretty sure that it is protonvpn's problem, waiting for them to respond.

Thanks and Enjoy,

PAM

**Comment by PalmerAL** [comment]
Yeah, if this an issue across all apps on your computer, it's probably their issue. But if it's only happening in Min and not other browsers, let me know.
